python/tvm/relay/backend/_backend.py

```diff
@@ -90,7 +90,11 @@ def _tensor_value_repr(tvalue):
 
 @tvm._ffi.register_func("relay._constant_repr")
 def _tensor_constant_repr(tvalue):
-    return str(tvalue.data.asnumpy())
+    # TODO(gus) do this in a smarter way
```

Comment:
Any ideas of how to do this better? Otherwise we can delete this comment.

Reply:
we decided that displaying data like this is an easy way with low overhead and gives correct information to users. Could be changed in the future if the need from users arises